projects/plugins/jetpack/modules/masterbar/admin-menu/class-wpcom-admin-menu.php

```diff
@@ -9,6 +9,8 @@
 
 use Automattic\Jetpack\Status;
 
+require_once __DIR__ . '/class-admin-menu.php';
```

Comment:
Maybe we should also remove `require_once __DIR__ . '/masterbar/admin-menu/class-admin-menu.php';` from the wpcom side `wp-content/mu-plugins/masterbar.php`?

Reply:
Yup, I was planning to do that in a separate diff after landing D56127-code (so changes between this PR and D56127-code are kept in sync).